test/SIL/ownership-verifier/load_borrow_invalidation_partial_apply.sil

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-sil-opt -enable-sil-verify-all -inline %s -verify-continue-on-failure=true -o /dev/null 2>&1 | %FileCheck %s
+// RUN: %target-sil-opt -enable-sil-verify-all -inline %s -verify-continue-on-failure=true -dont-abort-on-memory-lifetime-errors -o /dev/null 2>&1 | %FileCheck %s
 // REQUIRES: asserts
```

Comment:
What is broken in this test file?

Reply:
In addition to the existing error, the memory lifetime verifier also issues an error. The test would fail if the memory lifetime verifier aborts